vagrant/bootstrap.sh

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env bash
+# bootstrap script to install ff-node-monitor
+set -x
+
+# The name of your Freifunk community.
+INSTANCE_NAME = "Freifunk Kiel"
+# The sender address of the emails that are sent by ff-node-monitor.
+EMAIL_FROM="freifunk@r.z11.de"
+# The URL to the hopglass nodes.json file.
+NODES_URL="https://map.freifunk.in-kiel.de/yanic/nodes.json"
+# The root URL where you will be hosting ff-node-monitor.
+ROOT_URL="http://localhost:8833"
+# URL the service will be available from the outside (external IP)
+EXTERNAL="$1"
+
+# some variables used in this script
+WEB_URL="http://$EXTERNAL:8833"
```

Comment:
This does not seem suited for actual production use -- it means ports are visible to the user. Looks like this is for test setups only? If yes, it should say so somewhere very clearly.

Reply:
sure, this is a test-setup, but why would this be visible to the user?

Reply:
Because these URLs are sent out in emails.